Core/Interfaces/UI/API/Controllers/AbstractApiController.php

```diff
@@ -65,6 +74,20 @@ public function __construct(
         $this->queryBus          = $queryBus;
     }
 
+    /**
+     * Response with Cache-Control header
+     *
+     * @param mixed $data   Data
+     * @param int   $maxAge Cache max age
+     * @param int   $status Status code of response
+     *
+     * @return Response
+     */
+    public function cached($data, int $maxAge, int $status = 200) : Response
```

Comment:
так получается что можно кешировать только json, если переделать в 

```suggestion
    public function cached(Response $data, int $maxAge) : Response
```

тогда вызов будет выглядеть 
```php
return $this->cached($this->json($data, 200), Time.WEEK);
```

Reply:
Смысл этой обертки если оказалось решение из коробки $this->json($data)->setMaxAge($maxAge) выглядит красивее и понятнее